src/switch_joystick.c

```diff
@@ -107,24 +107,29 @@ static _GLFWmapping s_switchMapping =
     },
 };
 
+PadState pad;
+
 void _glfwInitSwitchJoysticks(void)
 {
     _GLFWjoystick* js = _glfwAllocJoystick(s_switchMapping.name, s_switchMapping.guid,
         _SWITCH_AXIS_COUNT, _SWITCH_BUTTON_COUNT, _SWITCH_HAT_COUNT);
 
+    padConfigureInput(1, HidNpadStyleSet_NpadStandard);
+    padInitializeDefault(&pad);
+
     js->mapping = &s_switchMapping;
-    js->nx.id = CONTROLLER_P1_AUTO;
+    padIsNpadActive (&pad, js->nx.id);
```

Comment:
This does nothing. Might want to remove this like, together with the "id" member (which doesn't really have any purpose anymore).